firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -13,3 +13,23 @@ bool alignPiece(int col, int row)
     return true;
 }
 
+// Assumes that the starting position is the center of the square the piece is on
+void centerPiece()
+{
+    // Loop counter
+    int i;
+
+    moveToFirstCircle();
+
+    // Enable electromagnet
+    analogWrite(ELECTROMAGNET, PWM_SCALE);
+
+    for (i = 0; i < NUM_CIRCLES; i++)
+    {
+        makeCircle(i);
```

Comment:
`makeCircle(i % 4)` and then you don't need to do `% 4` in the function

Reply:
Agreed

Reply:
If you do that, it would be a good idea to explain that getting an argument of 0, 1, 2, 3 is required

Reply:
The functions do need to know both the circle number and the starting quadrant. I changed it to pass both i and i%4 as parameters, but I'm not going to mark this conversation as resolved until you guys have had a chance to look at it.

Reply:
Looks great to me

Reply:
If it needs to know both then it's probably less complicated to just pass in the circle number. That way the function always accounts for bad inputs, plus it reduces the number of args needed to pass in. I don't think it's super important to pass in 
i % 4 for the sake of adding an extra arg.

Reply:
So the way it works now is probably fine

Reply:
Just kidding Nashir is right as always :)